grakn-test-tools/pom.xml

```diff
@@ -69,7 +69,12 @@
             <artifactId>embedded-redis</artifactId>
             <version>${redis-test.version}</version>
         </dependency>
-        
+        <dependency>
+            <groupId>com.github.zxl0714</groupId>
+            <artifactId>redis-mock</artifactId>
```

Comment:
Maybe it's worth switching to a mocked redis completely for our tests rather than the fake embedded redis we use for some of the tests? Do you think this mock can handle those cases? If so we should make that change.